WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Product Details/AggregatedProductListViewController.swift

```diff
@@ -0,0 +1,148 @@
+import UIKit
+import Yosemite
+
+
+/// A simple Product List presented from the Order Details screen -> Shipping Labels Packages.
+///
+/// The list shows the product name and the quantity.
+///
+final class AggregatedProductListViewController: UIViewController {
+    @IBOutlet private var tableView: UITableView!
+
+    var viewModel: OrderDetailsViewModel!
+    private var products: [Product]? = []
+    var items: [AggregateOrderItem] = []
+
+    private let imageService: ImageService = ServiceLocator.imageService
+
+    // MARK: - View Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.products = viewModel.products
+
+        configureMainView()
+        configureTableView()
+    }
+}
+
+
+// MARK: - Configuration
+//
+private extension AggregatedProductListViewController {
+
+    /// Setup: Main View
+    ///
+    func configureMainView() {
+        title = String.pluralize(items.count, singular: Localization.titleSingular, plural: Localization.titlePlural)
```

Comment:
Is it ok to group the items count in the here and in package cell, when we actually shipped more items? 😅 

4 items | 22 items
---- | ----
<img width="421" alt="Screen Shot 2021-06-17 at 8 46 09 AM" src="https://user-images.githubusercontent.com/562080/122411873-bbc96680-cf4a-11eb-929c-e3b9639e80de.png"> | <img width="401" alt="Screen Shot 2021-06-17 at 8 46 19 AM" src="https://user-images.githubusercontent.com/562080/122411884-bcfa9380-cf4a-11eb-9f95-42f75aef760d.png">

Reply:
I'm not sure about the approach that we want to use here. cc @Garance91540 @adamzelinski 
By the way, I added this header on top of the detail screen, because it's probably not clear what the number on the right indicates.

<img src="https://user-images.githubusercontent.com/495617/122424364-f35a0200-cf8e-11eb-82d4-0a6bd3a6296b.png" width=300 />

Reply:
Thank you @Garance91540, I updated the logic 👍